README.md

```
@@ -1,138 +1,12 @@
-## WebGPU tiny example
+## Caterfoil - 4D objects rendering on 2D screen
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)